src/librustdoc/html/static/storage.js

```diff
@@ -1,6 +1,9 @@
 // From rust:
 /* global resourcesSuffix */
 
+// eslint doesn't understand the functions here are used elsewhere
+/*eslint no-unused-vars: "off"*/
```

Comment:
Can you add spaces to pad this statement, so it can match the other lint statements?